tox.ini

```
@@ -9,39 +9,32 @@ deps =
 parallel_show_output = true
 usedevelop = true
 
-[testenv:black-python2.7]
-basepython = python3.7
-commands =
-    black --target-version=py27 --check pydruid setup.py tests
-deps =
-    black[python2]==19.10b0
-
-[testenv:black-python3.7]
-basepython = python3.7
+[testenv:black]
 commands =
     black --target-version=py37 --check pydruid setup.py tests
 deps =
-    black==19.10b0
+    black==22.3.0
 
-[testenv:{py27,py37}-isort]
+[testenv:isort]
 commands =
     isort --check-only --recursive pydruid setup.py tests
 deps =
      isort==4.3.21
 
-[testenv:{py27,py37}-flake8]
+[testenv:flake8]
 commands =
     flake8 pydruid setup.py tests
 deps =
     flake8==3.9.1
 
 [tox]
 envlist =
-    black-{python2.7,python3.7}
-    {py27,py37}-{isort,flake8,pytest}
+    black
+    flake8
+    isort
+    pytest
 skipdist = true
 
 [gh-actions]
 python =
-    2.7: py27-{isort,flake8,pytest}
-    3.7: black-python2.7,black-python3.7,py37-{isort,flake8,pytest}
+    3.7: py37
```

Hmm, the tests do not seem to have been executed.

Maybe it needs to be:

```suggestion
    3.7: black,flake8,isort,pytest
```

---

Well, i really doubted that line, it seemed very odd, but since i dont know anything about this, never done it before, it blindly trusted Mauri

---

:see_no_evil: